### PR TITLE
fix: pre-deployment checks work with fresh deployment

### PR DESCRIPTION
**Pull request by maghirardelli**
Issue #, if available:

Description of changes: Added check to only run the internal auth upgrade readiness check if the install was not a fresh deployment

Checklist:

- [x] Have you successfully deployed to an AWS account with your changes?
- [x] Have you successfully tested with your changes locally?

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: addons/addon-internal-auth-dep-check/packages/internal-auth-dep-check/lib/steps/internal-auth-dep-check-service.js [hunk 1]
```diff
@@ -42,6 +42,20 @@ class InternalAuthDepCheckService extends Service {
   async execute() {
     // Keep track of resources blocking upgrade
     const blockers = {};
+    const usersTable = this.settings.get(settingKeys.usersTableName);
+
+    // check not fresh install
+    try {
```
Comment (SanketD92) [inline addons/addon-internal-auth-dep-check/packages/internal-auth-dep-check/lib/steps/internal-auth-dep-check-service.js:48]: Could you also add a unit test for this case?

Comment (SanketD92) [inline addons/addon-internal-auth-dep-check/packages/internal-auth-dep-check/lib/steps/internal-auth-dep-check-service.js:48]: Could you also add a unit test for this case?

File: addons/addon-internal-auth-dep-check/packages/internal-auth-dep-check/lib/steps/internal-auth-dep-check-service.js [hunk 2]
```diff
@@ -42,6 +42,20 @@ class InternalAuthDepCheckService extends Service {
...
+      await this.dbService.dynamoDb.describeTable({ TableName: usersTable }).promise();
+    } catch (e) {
+      if (e.code === 'ResourceNotFoundException') {
+        this.log.info('This is first time deployment, no resources exist.');
+        return [];
```
Comment (SanketD92) [inline addons/addon-internal-auth-dep-check/packages/internal-auth-dep-check/lib/steps/internal-auth-dep-check-service.js]: Why return empty list? (why not simply `return;` ?)

Comment (maghirardelli) [reply addons/addon-internal-auth-dep-check/packages/internal-auth-dep-check/lib/steps/internal-auth-dep-check-service.js]: following the convention in the other pre-dep check! I can simply `return;` though!

**Review by nguyen102 — APPROVED** [review]

**Review by SanketD92 — APPROVED** [review]

**Pull request merged**